tests/drivers/display/display_read_write/testcase.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2024 (c) TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+common:
+  tags:
+    - drivers display
+  filter: dt_chosen_enabled("zephyr,display")
+  timeout: 1000
```

Comment:
Why the extra long timeout? (this line seems not needed in this case as anyhow the tests are not run)

Reply:
I worry that this module sometimes delays displaying the window.
But this test is `build_only` at this time, so remove this.